From fcc1af1aab769ac494bdec1476bbb9f45bfbee6b Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 30 Jan 2018 15:36:50 +0800
Subject: [PATCH 3300/5242] MLK-17473-6 drm/imx: dcss: remove
 'allow_fb_modifiers' assignment

commit  d064106026024d5ac8a7d069ea34c93ff1829841 from
https://source.codeaurora.org/external/imx/linux-imx.git

Since the 'allow_fb_modifiers' has been assigned to be true in
imx drm core driver in bind(), it is not necessary to set this
flag again here.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/dcss/dcss-crtc.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/imx/dcss/dcss-crtc.c b/drivers/gpu/drm/imx/dcss/dcss-crtc.c
index 24cd881..c8e0cf6 100644
--- a/drivers/gpu/drm/imx/dcss/dcss-crtc.c
+++ b/drivers/gpu/drm/imx/dcss/dcss-crtc.c
@@ -310,8 +310,6 @@ static int dcss_crtc_init(struct dcss_crtc *crtc,
 		return ret;
 	}
 
-	drm->mode_config.allow_fb_modifiers = true;
-
 	disable_irq(crtc->irq);
 
 	return 0;
-- 
1.7.9.5

